packages/amplify_api/example/tool/provision_integration_test_resources.sh

```diff
@@ -2,6 +2,8 @@
 set -e
 IFS='|'
 
+[ "$AWS_PROFILE" ] && profileName="$AWS_PROFILE" || profileName="default";
```

Comment:
```suggestion
profileName=${AWS_PROFILE:-default}
```

Reply:
good suggestion, updated here and other scripts